files/jcl/ZWESECUR.jcl

```diff
@@ -225,39 +249,52 @@
 /* show results .................................................... */
   LISTGRP  &STCGROUP. OMVS
   LISTUSER &ZOWEUSER. OMVS
-  LISTUSER &XMEMUSER.  OMVS
+  LISTUSER &XMEMUSER. OMVS
   LISTUSER &AUXUSER.  OMVS
   RLIST STARTED &ZOWESTC..* ALL STDATA
-  RLIST STARTED &XMEMSTC..*  ALL STDATA
+  RLIST STARTED &XMEMSTC..* ALL STDATA
   RLIST STARTED &AUXSTC..*  ALL STDATA
 
 /* DEFINE ZOWE SERVER PERMISIONS ................................... */
 
-/* permit Zowe main server to use XMEM, cross memory server          */
+/* permit Zowe main server to use XMEM cross memory server           */
   RLIST   FACILITY ZWES.IS ALL
   RDEFINE FACILITY ZWES.IS UACC(NONE)
   PERMIT ZWES.IS CLASS(FACILITY) ACCESS(READ) ID(&ZOWEUSER.)
 
   SETROPTS RACLIST(FACILITY) REFRESH
 
+/* permit Zowe main server to create a user's security environment   */
+/* ATTENTION: Defining the BPX.DAEMON or BPX.SERVER profile makes    */
+/*            z/OS UNIX switch to z/OS UNIX level security. This is  */
+/*            more secure, but it can impact operation of existing   */
+/*            applications. Test this thoroughly before activating   */
+/*            it on a production system.                             */
+  RLIST   FACILITY BPX.SERVER ALL
+  RDEFINE FACILITY BPX.SERVER UACC(NONE)
+  PERMIT BPX.SERVER CLASS(FACILITY) ACCESS(UPDATE) ID(&ZOWEUSER.)
+
```

Comment:
If I understand correctly, &ZOWEUSER should also have UPDATE access to BPX.DAEMON. 
`PERMIT BPX.DAEMONCLASS(FACILITY) ACCESS(UPDATE) ID(&ZOWEUSER.)`

Reply:
The original security setup for ZSS reauired XMEMUSER to have BPX.DAEMON and BPX.ZERVER. IT was ZOWEUSER that was missing since we were hijacking zOSMF's USERID, which already had it.